### PR TITLE
DM-23801: fix the Contributing entry in index.rst and check that code is formatted with black

### PR DESCRIPTION
**Pull request by r-owen**

### CODE REVIEW

File: doc/revision_history.rst [hunk 1]
```diff
@@ -6,6 +6,36 @@
 ts_ATDomeTrajectory Revision History
```
Comment (couger01) [inline doc/revision_history.rst:6]: Just a pedantic note, could you name this release notes instead of revision history?

**Review by couger01 — CHANGES_REQUESTED** [review]
Just a change to consider. Otherwise looks good.

File: doc/revision_history.rst (hunk 1 above)
Comment (r-owen) [reply doc/revision_history.rst:6]: I already have about 10 packages that use `revision_history.rst`. I would rather not change them all (or at least would rather change them all on some other ticket if we're going to change this).

**Review by couger01 — APPROVED** [review]
LGTM

**Pull request merged**